socialapp/core/views.py

```diff
@@ -1,10 +1,32 @@
 from django.shortcuts import render
 from django.http import HttpResponse
-from .models import Country
+from .models import Post, Comment
 
 # Create your views here.
 
 
 def index(request):
     return HttpResponse('I think it works...')
 
+
+def posts_page(request):
+    posts = Post.objects.all().values('text', 'id')
+    comments = Comment.objects.all().values('text', 'post')
```

Comment:
To be more readable, I suggest to loop directly through posts and to get the comments using the related name. So, don't need the extra variable `post_entries`.

Reply:
I agree. In the next issue I will remove a big part from this code and I will send just the posts variable via context to the template in order to manage the comments.